opensearch-mixin/config.libsonnet

```diff
@@ -1,8 +1,9 @@
 {
   _config+:: {
+    enableMultiCluster: false,
     // extra static selector to apply to all templated variables and alerts
-    filteringSelector: 'cluster!=""',
-    groupLabels: ['job', 'cluster'],
+    filteringSelector: if self.enableMultiCluster then 'opensearch_cluster!="",cluster!=""' else 'opensearch_cluster!=""',
+    groupLabels: if self.enableMultiCluster then ['job', 'opensearch_cluster', 'cluster'] else ['job', 'opensearch_cluster'],
```

Comment:
I believe you would want opensearch_cluster to come after cluster since the hierarchy goes from overarching down.

Reply:
so something like 
```suggestion
    groupLabels: if self.enableMultiCluster then ['job', 'cluster', 'opensearch_cluster'] else ['job', 'opensearch_cluster'],
```